src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -32,9 +32,9 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
         private readonly IErrorListInfoBarController errorListInfoBarController;
         private readonly ISolutionBindingInformationProvider solutionBindingInformationProvider;
 
-        public event EventHandler<bool> SolutionBindingChanged;
+        public event EventHandler<ActiveSolutionBinding> SolutionBindingChanged;
 
-        public bool IsActiveSolutionBound { get; private set; }
+        public ActiveSolutionBinding ActiveSolutionBinding { get; private set; } = new ActiveSolutionBinding();
```

Comment:
Ah, ok - if you rename the class to _XXXEventArgs_, then it will look odd as a property type.
You could just have two properties on this class, one for the "is bound" and one for the project id.